preview-src/pullRequestOverviewRenderer.ts

```diff
@@ -135,13 +136,18 @@ export function renderStatusChecks(pr: PullRequest) {
 
 		statusElement.appendChild(state);
 
-		const statusIcon = renderUserIcon(s.url, s.avatar_url);
+		const statusIcon = renderUserIcon(s.target_url, s.avatar_url);
```

Comment:
I merged the timeline events PR so you may need to convert the casing.

Reply:
I think it's actually ok right now, only the REST API is being used to get status checks, it isn't part of the timeline events call